c7n/resources/glue.py

```diff
@@ -205,3 +206,88 @@ def process(self, resources):
                 client.delete_crawler(Name=r['Name'])
             except client.exceptions.EntityNotFoundException:
                 continue
+
+
+@resources.register('glue-database')
+class GlueDatabase(QueryResourceManager):
+
+    class resource_type(object):
+        service = 'glue'
+        enum_spec = ('get_databases', 'DatabaseList', None)
+        detail_spec = None
+        id = name = 'Name'
+        date = 'CreatedOn'
+        dimension = None
+        filter_name = None,
+        type = 'database'
+        state_key = 'State'
+        arn = None
+
+    permissions = ('glue:Getdatabases',)
```

Comment:
generally resource manager classes we can auto construct permissions from the method names, plus in this case the manually specified permission isn't valid.